privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -357,23 +366,14 @@ def attack(self):
         )
 
         predictions = {}
-        # iterate over unbalanced ratios in 0.05 steps (0.05-0.45, 0.55-0.95)
+        # iterate over unbalanced ratios
         # (e.g. 0.55 means: class 0: 0.45 of all samples, class 1: 0.55 of all samples)
 
-        for ratio in np.arange(0.55, 1, 0.05):
+        for ratio in self.ratios_for_attack:
             # goes through ratios 0.55 - 0.95
```

Comment:
this comment isn't up to date. It depends on given list of ratios

Reply:
Changed in new pull request so I resolved the conversation.